filters/flowid/hashing.go

```diff
@@ -0,0 +1,37 @@
+package flowid
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"regexp"
+)
+
+const (
+	defaultLen = 16
+	maxLength  = 254
+	minLength  = 8
+)
+
+var (
+	ErrInvalidLen  = errors.New(fmt.Sprintf("Invalid length. len must be >= %d and < %d", minLength, maxLength))
+	flowIdRegex, _ = regexp.Compile(`^[\w+/=\-]+$`)
+)
+
+func newFlowId(len uint8) (string, error) {
+	if len < minLength || len > maxLength || len%2 != 0 {
```

Comment:
if it works, it works. just a warning that i managed once to trigger very strange behavior from the compiler by overdefining 'make' and it resulted in a relatively hard-to-figure compilation error.